test/operatortest.jl

```diff
@@ -10,11 +10,24 @@
 
 @assert coeffs(commutator(sigma_x, sigma_x)) == spzeros(2,2)
 
-######################################
-# Position & Momentum Operators Test #
-######################################
+####################################################
+# Position, Displacement & Momentum Operators Test #
+####################################################
 
 p = positionop(2)
 m = momentumop(2)
 @assert coeffs(commutator(sigma_x, p)) == spzeros(2,2)
 @assert coeffs(commutator(sigma_y, m)) == spzeros(2,2)
+
+csvd = coherentstatevec_inf(20,1)
+csvo = displaceop(20,1)*statevec(1,FiniteBasis(20))
```

Comment:
Maybe call `coherentstatevec` instead? It would be more readable and the definition is the same.